docker-compose.yml

```diff
@@ -36,8 +36,10 @@ services:
       MYSQL_DATABASE: project_a
       MYSQL_USER: project_a
       MYSQL_PASSWORD: ${DATABASE_PASSWORD}
+      MYSQL_TCP_PORT: ${DATABASE_PORT}
+
     ports:
-      - 3306:3306 # for debug
+      - ${DATABASE_PORT}:${DATABASE_PORT} # for debug
```

Comment:
Je pense que c'est mieux un ${DATABASE_PORT}:3306
Ca évite les soucis de vins sur le container

Reply:
Keycloak database will not get exposed even for debug (unless extreme desperation) since it's already hard to read.